src/main/java/org/opensearch/security/tools/democonfig/Installer.java

```diff
@@ -205,23 +205,28 @@ static void setOpenSearchVariables() {
         OPENSEARCH_LIB_PATH = BASE_DIR + "lib" + File.separator;
         OPENSEARCH_INSTALL_TYPE = determineInstallType();
 
+        boolean shouldExit = false;
         if (!(new File(OPENSEARCH_CONF_FILE).exists())) {
-            System.out.println("Unable to determine OpenSearch config directory. Quit.");
-            System.exit(-1);
+            System.out.println("Unable to determine OpenSearch config file. Quit.");
```

Comment:
Are we should be running the same checkstyle + spotless rules on these files as the rest of the codebase?

Reply:
I see that you disagreed. Other files in tools package use the same format of using sysout to print information. To address this, we added an except to checkstyle [here](https://github.com/opensearch-project/security/blob/main/checkstyle/checkstyle.xml#L41)